pull_request_template.md

```diff
@@ -1,22 +0,0 @@
-## Description
```

Comment:
We had `.github/pull_request_template.md` so bringing this to that file instead of duplicating it (that file takes precedence over this)